qtodotxt/lib/tasklib.py

```diff
@@ -2,10 +2,27 @@
 import re
 
 from PyQt5 import QtCore
-
+from enum import Enum
 from qtodotxt.lib.task_htmlizer import TaskHtmlizer
 
 
+class recursiveModes(Enum):
+    init = 0     # Initial recursive mode
```

Comment:
btw, maybe it is cleaner to use RecursiveMode.None instead og setting class member to None. I let you decide.